config/gaiatest_testvars.json

```diff
@@ -101,6 +101,13 @@
 		"device_cert": "cert_device",
 		"branch_cert": "v2.0",
 		"buildname_cert": "cert_build-.master.B-xxx.Gecko-xxxxxxx.Gaia-xxxxxxx"
+	},
+	"visualtest": {
+		"enabled": true,
+		"engine": "pil",
+		"save_baseline": false,
+		"baseline_dir": "",
+		"cmp_dir": ""
```

Comment:
I wanted to discuss something first with you, before updating them. There will be a second PR to deal with it